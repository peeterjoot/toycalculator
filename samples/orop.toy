//THIS IS A GENERATED TEST CASE (./bitwiseop.perl --or).  DO NOT EDIT

STRING s[16];
INT64 r;
BOOL i1;
INT16 i16;
INT32 i32;
INT64 i64;
INT8 i8;
BOOL j1;
INT16 j16;
INT32 j32;
INT64 j64;
INT8 j8;
i1 = TRUE;
i16 = 1000;
i32 = 100000;
i64 = 100000000000;
i8 = 10;
j1 = FALSE;
j16 = 100;
j32 = 10000;
j64 = 10000000000;
j8 = 1;
s = "i1 OR j1";
PRINT s;
r = i1 OR j1;
PRINT r;
s = "i1 OR j16";
PRINT s;
r = i1 OR j16;
PRINT r;
s = "i1 OR j32";
PRINT s;
r = i1 OR j32;
PRINT r;
s = "i1 OR j64";
PRINT s;
r = i1 OR j64;
PRINT r;
s = "i1 OR j8";
PRINT s;
r = i1 OR j8;
PRINT r;
s = "i16 OR j1";
PRINT s;
r = i16 OR j1;
PRINT r;
s = "i16 OR j16";
PRINT s;
r = i16 OR j16;
PRINT r;
s = "i16 OR j32";
PRINT s;
r = i16 OR j32;
PRINT r;
s = "i16 OR j64";
PRINT s;
r = i16 OR j64;
PRINT r;
s = "i16 OR j8";
PRINT s;
r = i16 OR j8;
PRINT r;
s = "i32 OR j1";
PRINT s;
r = i32 OR j1;
PRINT r;
s = "i32 OR j16";
PRINT s;
r = i32 OR j16;
PRINT r;
s = "i32 OR j32";
PRINT s;
r = i32 OR j32;
PRINT r;
s = "i32 OR j64";
PRINT s;
r = i32 OR j64;
PRINT r;
s = "i32 OR j8";
PRINT s;
r = i32 OR j8;
PRINT r;
s = "i64 OR j1";
PRINT s;
r = i64 OR j1;
PRINT r;
s = "i64 OR j16";
PRINT s;
r = i64 OR j16;
PRINT r;
s = "i64 OR j32";
PRINT s;
r = i64 OR j32;
PRINT r;
s = "i64 OR j64";
PRINT s;
r = i64 OR j64;
PRINT r;
s = "i64 OR j8";
PRINT s;
r = i64 OR j8;
PRINT r;
s = "i8 OR j1";
PRINT s;
r = i8 OR j1;
PRINT r;
s = "i8 OR j16";
PRINT s;
r = i8 OR j16;
PRINT r;
s = "i8 OR j32";
PRINT s;
r = i8 OR j32;
PRINT r;
s = "i8 OR j64";
PRINT s;
r = i8 OR j64;
PRINT r;
s = "i8 OR j8";
PRINT s;
r = i8 OR j8;
PRINT r;
