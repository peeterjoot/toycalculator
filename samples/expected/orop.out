12340000
1
12340001
101
12340002
10001
12340003
10000000001
12340004
1
12340005
1000
12340006
1004
12340007
10232
12340008
10000001000
12340009
1001
12340010
100000
12340011
100068
12340012
108464
12340013
10000000672
12340014
100001
12340015
100000000000
12340016
100000000100
12340017
100000001808
12340018
100336135168
12340019
100000000001
12340020
10
12340021
110
12340022
10010
12340023
10000000010
12340024
11
